android/accessibility_guide.md

```diff
@@ -0,0 +1,32 @@
+# Android Accessibility Guide
+Accessibility (or "a11y") is about making apps usable by everyone. Android developers need to pay particular attention to users with visual impairment, auditory impairment, and restricted motor skills. This guide is intended to get you started with making your Android app more accessible.
+
+To identify how well the accessibility of your app is working, try it as a user! For example, try closing your eyes while using the TalkBack screen reader. You can also use [the Accessibility Scanner][scanner] for an automated analysis.
+
+If you have questions, please ask!
+
+## Introductory resources
+The [official Android accessibility overview](https://developer.android.com/guide/topics/ui/accessibility/apps) is a great overview to common issues users may face and how you should address them as an Android developer. A good supplement, for visual impairment, is [this official Android Developers video](https://www.youtube.com/watch?v=1by5J7c5Vz4). This [presentation on accessibility](https://www.youtube.com/watch?v=UOr3mgqJU0A&feature=youtu.be&list=PLnVy79PaFHMUqqvwbjyKJZv1N8rzHOCBi) and [this podcast](https://androidbackstage.blogspot.com/2014/10/episode-14-accessibility.html) are also recommended by our developers.
+
+Some general tips:
+- Look through the accessibility preferences in the Developer Options: there are many features, such as visual simulations, to help you.
+- If it is impossible to provide an intuitive, accessible user experience for your current UI, this is a design smell: a UI that is accessible tends to be the best experience for all users.
+    - For example, a dialog with one option, like "Remove", that is dismissed by clicking outside the dialog, will be unintuitive to screen reader users. However, this dialog may be unintuitive to users who are unfamiliar with common software paradigms (i.e. tap outside to dismiss) too. A better dialog would be one with a "dismiss" option in addition to "remove"
+- Avoid creating separate code paths and user experiences to address accessibility issues: in our experience, accessibility is not often tested so it's easy to break the experience if you forget to update one code path
+
+## Screen reader tips
```

Comment:
Might be worth adding an aside about Dark Screen; on iOS I've used a similar function called ScreenCurtain. Having the screen blacked out with VoiceOver on is an awesome way to test your interface!!

Reply:
@sashei Is Dark Screen an app? Or is it a general suggestion to find a way to black out the screen content? Neat idea. :)

Reply:
Ohh, I have done some more digging and it's only on Samsung phones: https://www.samsung.com/us/support/answer/ANS00062641/